src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <Compile Include="$(CommonTestPath)\System\TypeBuilderExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
+      <CodeCoverageAssemblies Include="Microsoft.CSharp" /> 
```

Comment:
@mellinoe you are correct I did miss that it is a property in our CodeCoverage.targets file. I would suggest turning that into an item group though and avoid using the properties that start with underscore in project files. However @JonHanna  keep what you have for your PR as I filed https://github.com/dotnet/corefx/issues/15429 to follow-up on that aspect.